modules/tools/debugserver/openocd/mak/Makefile

```diff
@@ -42,10 +44,48 @@ OPENOCD_BIN 		   = openocd
 OPENOCD_FLAGS 		   = -f $(OPENOCD_CFG)
 
 # openocd dependent makefile
--include $(OPENOCD_PATH)$(DS)mak$(DS)$(ARCH)$(DS)$(CPUTYPE)$(DS)$(CPU)$(DS)Makefile
+include $(OPENOCD_PATH)$(DS)mak$(DS)$(ARCH)$(DS)$(CPUTYPE)$(DS)$(CPU)$(DS)Makefile
```

Comment:
ok! lo vuelvo a agregar!


El 09/07/2016 a las 08:26 p.m., Juan Cecconi escribió:
>
> In modules/tools/debugserver/openocd/mak/Makefile 
> <https://github.com/ciaa/Firmware/pull/431#discussion_r70173562>:
>
> > @@ -42,10 +44,48 @@ OPENOCD_BIN 		   = openocd
> >  OPENOCD_FLAGS 		   = -f $(OPENOCD_CFG)
> >
> >  # openocd dependent makefile
> > --include $(OPENOCD_PATH)$(DS)mak$(DS)$(ARCH)$(DS)$(CPUTYPE)$(DS)$(CPU)$(DS)Makefile
> > +include $(OPENOCD_PATH)$(DS)mak$(DS)$(ARCH)$(DS)$(CPUTYPE)$(DS)$(CPU)$(DS)Makefile
>
> El - hay que dejarlo para los casos que el makefile de esa ARCH no 
> esté porque no aplica openocd, indica que puede no estar el archivo y 
> no tiene que dar error
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/ciaa/Firmware/pull/431/files/615d43ab217be7022671da1fe278e3051920aed3..f72c1b7f12d6c5fa22b0514926f5dc78378ca594#r70173562>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AQg2Fs11zHFv7JxNOd08eqgUU7VtncwKks5qUC46gaJpZM4JFml7>.
>